policyengine_us/parameters/gov/states/mt/tax/income/credits/rebate/property/amount.yaml

```
@@ -1,6 +1,7 @@
 description: Montana provides the following property tax rebate amount.
 values:
   2022-01-01: 675
+  2024-01-01: 0 #was only for 2022 and 2023
```

This is already reflected in the `non_refundable.yaml` file in the montana parameters we dont need to 0 out the parameter value as it technically is not accurate (the parameter is not set to 0 - the program is no longer in effect) 
```suggestion
  # The property tax rebate was only offered in the 2022 and 2023 tax years 
```